Add dashboard sharing (#63)
axiomhq/docs: query-data/dashboards.mdx

Click **Save Dashboard** to save your changes.

## Share dashboards

To specify who can access a dashboard:

1. In the toolbar, click <img src="/doc-assets/icons/share.svg" className="inline-icon" alt="Share" /> **Share**.
2. Select one of the following:
    - Select **Just Me** to make the dashboard private. Only you can access the dashboard.
    - Select a group in your Axiom organization. Only members of the selected group can access the dashboard. For more information about groups, see [Access](/reference/settings#access-overview).

>     - Select **Everyone** to make the dashboard public. Everyone in your Axiom organization can access the dashboard.

Reviewer: I think public is the wrong choice of words. It risks a misunderstanding that it is accessible to anybody on the web. But _not_ a blocking concern for this PR

Author: Good point! I've updated this.

>     - Select **Everyone** to make the dashboard public. Everyone in your Axiom organization can access the dashboard.

Author: ```suggestion
    - Select **Everyone** to make the dashboard accessible to all users in your Axiom organization.
```
